site/src/pages/RoadmapPage/sidebar/ProgramRequirementsList.tsx

```diff
@@ -307,12 +372,17 @@ const ProgramRequirementsList: FC<RequireCourseListProps> = ({ requirements, sto
     .flatMap((quarter) => quarter.courses)
     .map((course) => [course.id, course.minUnits]);
   const transferCourseMap = transferredCourses.map((t) => [t.courseName.replace(/\s/g, ''), t.units ?? 0]);
+  const transferCourseMapWithType = transferredCourses.map((t) => [
+    t.courseName.replace(/\s/g, ''),
+    { units: t.units ?? 0, transferType: t.transferType },
+  ]);
```

Comment:
as per previous comment:

```suggestion
    .map((course) => [course.id, { units: course.minUnits }]);
  const transferCourseMap = transferredCourses.map((t) => [
    t.courseName.replace(/\s/g, ''),
    { units: t.units ?? 0, transferType: t.transferType },
  ]);
```